apps/namadillo/src/App/Settings/SettingsMain.tsx

```diff
@@ -11,19 +19,65 @@ const { VITE_REVISION: revision = "" } = import.meta.env;
 export const SettingsMain = (): JSX.Element => {
   const indexerHealth = useAtomValue(indexerHeartbeatAtom);
   const maspIndexerHealth = useAtomValue(maspIndexerHeartbeatAtom);
+  const settingsMutation = useAtomValue(updateSettingsProps);
+  const settings = useAtomValue(settingsAtom);
 
   return (
     <div className="flex flex-1 justify-between flex-col w-full">
       <ul className="flex flex-col gap-2">
-        <SettingsPanelMenuItem url={routes.settingsAdvanced} text="Advanced" />
-        <SettingsPanelMenuItem
-          url={routes.settingsSignArbitrary}
-          text="Sign Arbitrary"
-        />
+        {settings.advancedMode && (
+          <SettingsPanelMenuItem
+            url={routes.settingsAdvanced}
+            text="Advanced"
+          />
+        )}
+        {settings.advancedMode && (
+          <SettingsPanelMenuItem
+            url={routes.settingsSignArbitrary}
+            text="Sign Arbitrary"
+          />
+        )}
         <SettingsPanelMenuItem url={routes.settingsMASP} text="MASP" />
         <SettingsPanelMenuItem url={routes.settingsLedger} text="Ledger" />
       </ul>
       <div className="text-xs">
+        <div className="relative mb-4 flex items-center gap-2 max-w-full">
+          <ToggleButton
+            onChange={() => {
+              settingsMutation.mutateAsync({
+                key: "advancedMode",
+                value: !settings.advancedMode,
+              });
+            }}
+            label="PRO mode"
+            checked={settings.advancedMode}
+            activeColor="yellow"
+            color="white"
+            containerProps={{ className: "[&_span]:order-3 gap-3" }}
+          />
+          <span className="text-yellow relative">
+            <IconTooltip
+              icon={<FaInfo />}
+              text={
+                <>
+                  PRO mode unlocks more technically advanced features in
+                  Namadillo that replicates the UX of the CLI. You can learn
+                  more about these features{" "}
+                  <a
+                    href="https://docs.namada.net/users"
+                    target="_blank"
+                    rel="nofollow noreferrer"
+                  >
+                    here
```

Comment:
The word `here` is a little bit off because the parent has `display:flex`, maybe wrapping the content in a `div` can fix this 
![Screenshot 2025-03-10 at 19 42 44](https://github.com/user-attachments/assets/7a76e280-3908-42b0-9414-78634c036ed6)

Reply:
we could also add a `text-yellow hover:underline` or similar to highlight the link